### ISSUE TITLE
Not working at IE11 with latest MobX 6

### ISSUE THREAD

**Issue by Dev-Taehui** [mobxjs/mobx#3771]
**Intended outcome:** Working at IE11

**Actual outcome:** Not working at IE11 and throws exception (@@toPrimitive must return a primitive value.)

**How to reproduce the issue:** I created minimal reproducible github project (https://github.com/Dev-Taehui/ie11test).
Clone it, npm install, npm run local for build and serve to local test.

**Versions**
MobX 6.10.2
React 17.0.2
React-Scripts 4.0.3
Node.js 18.17.1
IE 11

**Comment by mweststrate** [comment]
What is the latest version that does work, to be able to intersect a potential cause? (I don't have IE11 at my disposal here)

**Comment by Dev-Taehui** [comment]
@mweststrate I tried MobX 6.3.3, MobX 6.0.0. But it doesn't work with same reason.

I tested IE11 with VBS script.
Create `Launch IE.vbs` file and write this command to it.

```
CreateObject("InternetExplorer.Application").Visible = true
```

Then, run it

**Comment by mweststrate** [comment]
I don't have Windows neither 🙂. But if older versions don't work as well,
ot might also be something else in your stack. Like the build tooling. What
made you conclude it is MobX? Do you have a stack for the error? Are the
React and create-react-app version etc IE 11 compatible?

**Comment by Dev-Taehui** [comment]
@mweststrate According to the example I posted, calling `mobx`'s `configure({ useProxies: "never" })` in index.js does not work in IE11. If I remove that code it works on IE11

**Comment by Dev-Taehui** [comment]
@mweststrate Does need other polyfills? Accroding to thrown error, It seems to be that @@toPrimitive Symbol and decorator was not fully polyfilled.

**Comment by mweststrate** [comment]
Yeah you will need some Symbol polyfills: https://github.com/zloirock/core-js#ecmascript-symbol. You could probably do a quick experiment with:

```
if (!window.Symbol) {
  window.Symbol = {}
}
if (!window.Symbol.toPrimitive) {
  window.Symbol.toPrimitive == "@@toPrimitive"
}
```

Just as experiment, it'll probably kill something else as it is not a valid implementation.
